server/src/main/java/org/apache/iotdb/db/service/IoTDB.java

```diff
@@ -88,7 +91,17 @@ private void setUp() throws StartupException {
     if (IoTDBDescriptor.getInstance().getConfig().isEnableStatMonitor()) {
       StatMonitor.getInstance().recovery();
     }
-
+    // Delete data/system/users/*.profile
+    File usersDir = SystemFileFactory.INSTANCE.getFile(
+        IoTDBDescriptor.getInstance().getConfig().getSystemDir() + File.separator + "users");
```

Comment:
It might be more convenient to use interface `File getFile(String parent, String child)`:
```suggestion
        IoTDBDescriptor.getInstance().getConfig().getSystemDir(), "users");
```

Reply:
Thanks for your suggestion. :)